core/keys/function_key.talon-list

```diff
@@ -0,0 +1,26 @@
+list: user.function_key
+-
+f one key: f1
```

Comment:
Please double check that community by default uses "f one key" instead of "f one" — thanks!